byos/readme.md

```diff
@@ -61,7 +61,9 @@ The following documents outline the permissions needed for each OpenHack, the mo
 - [AI-Powered Knowledge Mining](knowledge-mining/deployment.md)
 - [App Modernization with NoSQL](app-modernization-no-sql/deployment.md)
 - [Containers](containers/deployment.md)
-- [DevOps 2.0](devops-2.0/deployment.md)
+- DevOps
```

Comment:
I am not sure removing 2.0 is correct.  The number is how we do versioning. Could you confirm that we are not supposed to be in the 2.0 or 3.0 folder?  Are we modifying the correct path?